src/Discord.Net.WebSocket/Entities/Channels/SocketNewsChannel.cs

```diff
@@ -22,31 +27,61 @@ internal new static SocketNewsChannel Create(SocketGuild guild, ClientState stat
             entity.Update(state, model);
             return entity;
         }
+        /// <inheritdoc />
+        /// <remarks>
+        ///     <note type="important">
+        ///     This property is not supported by this type. Attempting to use this property will result in a <see cref="NotSupportedException"/>.
+        ///     </note>
+        /// </remarks>
```

Comment:
I think the not supported properties of this class would be a good fit for the `exception` tag.